Readme.md

```diff
@@ -44,7 +44,10 @@ Get install instructions for other combinations in the `Get Started Easily` sect
 
 3. Run the training script
 
-    - `python ./ort/tests/bert_for_sequence_classification.py`
+    - `python ./ort/samples/bert_for_sequence_classification.py`
+
+
+        > **NOTE:** To run the nightly sample test, run `python ./ort/`**`tests`**`/bert_for_sequence_classification.py` instead.
```

Comment:
Changed `Readme.md` to say `Verify` instead of test. Removed the `DebugOptions` from the test so the sample and test file are the same for the time being.